tests/tests_api/test_group.py

```diff
@@ -272,4 +287,79 @@ def test_public_group_start_teacher_status(
         teacher.refresh_from_db()
         assert teacher.groups.count() == teacher.simultaneous_groups + delta
         assert teacher.status == expected_status
-        self._compare_date_time_with_timestamp(teacher.status_since, timestamp)
+        compare_date_time_with_timestamp(teacher.status_since, timestamp)
+
+
+class TestPublicGroupAbort:
+    @staticmethod
+    def _make_url(group: Group) -> str:
+        return reverse("groups-abort", kwargs={"pk": group.id})
+
+    def test_public_group_abort_general_check(self, api_client, active_group, timestamp):
+        response = api_client.post(self._make_url(active_group))
+
+        assert response.status_code == status.HTTP_200_OK
+
+        active_group.refresh_from_db()
+        assert active_group.status == GroupStatus.ABORTED
+
+        common_status_since = active_group.status_since
+        compare_date_time_with_timestamp(common_status_since, timestamp)
+
+        assert active_group.students.count() == 0
```

Comment:
`assert not active_group.students.count()` is a little bit shorter, but not necessary to fix it, just more "python-way"

Reply:
same below